bcc-site/bcc/templates/part_detail.html

```diff
@@ -1,19 +1,38 @@
 {% extends "layout.html" %}
 {% block body %}
-
+<br>
+<br>
 <div class="row">
     <div class="col-sm-8">
         <h1>{{ part.partNumber}}</h1>
     </div>
     <div class="col">
-        {% if part.status == 'ACTIVE' %}
+        {% if part.status == 1 %}
```

Comment:
Part status would be better as states like 'ACTIVE'/etc. - better to avoid magic numbers